pkg/devfile/adapters/kubernetes/adapter.go

```diff
@@ -48,6 +48,17 @@ func (k Adapter) Build(parameters common.BuildParameters) error {
 	return nil
 }
 
+// Build creates Kubernetes resources to build an image for the component
+func (k Adapter) Deploy(parameters common.DeployParameters) error {
+
+	err := k.componentAdapter.Deploy(parameters)
+	if err != nil {
+		return errors.Wrap(err, "Failed to build image for the component")
```

Comment:
need to update err here, maybe we can just do a big commit to clean up everything like this

Reply:
Changed error message